third_party/closure-compiler/Messages.properties

```diff
@@ -0,0 +1,335 @@
+#
```

Comment:
from what i can tell they are not exported in the jar compilation, and they are needed for reporting AST errors during test.